src/app/utility/energy-consumption/energy-consumption.component.html

```diff
@@ -1,38 +1,38 @@
 <div class="tabs">
     <ul>
         <li><a routerLink="/utility/energy-consumption" [routerLinkActive]="['active']" [routerLinkActiveOptions]="{ exact: true }">Energy Source</a></li>
-        <li [hidden]="!electricity"[class.red]="!hasElectricityData">
+        <li [hidden]="!electricity" [class.red]="hasElectricityError">
             <a routerLink="electricity" [routerLinkActive]="['active']">Electricity 
                 <!-- <span [hidden]="hasElectricityData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!naturalGas" [class.red]="!hasNaturalGasData">
+        <li [hidden]="!naturalGas" [class.red]="hasNaturalGasError">
             <a routerLink="natural-gas" [routerLinkActive]="['active']">Natural Gas 
                 <!-- <span [hidden]="hasNaturalGasData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!otherFuels" [class.red]="!hasOtherFuelsData">
-            <a routerLink="other-fuels" [routerLinkActive]="['active']">Other Fuels 
+        <li [hidden]="!otherFuels">
+            <a routerLink="other-fuels" [routerLinkActive]="['active']" [class.red]="hasOtherFuelsError">Other Fuels 
                 <!-- <span [hidden]="hasOtherFuelsData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!otherEnergy" [class.red]="!hasOtherEnergyData">
-            <a routerLink="other-energy" [routerLinkActive]="['active']">Other Energy 
+        <li [hidden]="!otherEnergy">
+            <a routerLink="other-energy" [routerLinkActive]="['active']" [class.red]="hasOtherEnergyError">Other Energy 
                 <!-- <span [hidden]="hasOtherEnergyData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!water" [class.red]="!hasWaterData">
-            <a routerLink="water" [routerLinkActive]="['active']">Water 
+        <li [hidden]="!water">
+            <a routerLink="water" [routerLinkActive]="['active']" [class.red]="hasWaterError">Water 
                 <!-- <span [hidden]="hasWaterData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!wasteWater" [class.red]="!hasWasteWaterData">
-            <a routerLink="waste-water" [routerLinkActive]="['active']">Waste Water 
+        <li [hidden]="!wasteWater">
+            <a routerLink="waste-water" [routerLinkActive]="['active']" [class.red]="hasWasteWaterError">Waste Water 
                 <!-- <span [hidden]="hasWasteWaterData">*</span> -->
             </a>
         </li>
-        <li [hidden]="!otherUtility" [class.red]="!hasOtherUtilityData">
-            <a routerLink="other-utility" [routerLinkActive]="['active']">Other Utility 
+        <li [hidden]="!otherUtility">
+            <a routerLink="other-utility" [routerLinkActive]="['active']" [class.red]="hasOtherUtilityError">Other Utility 
                 <!-- <span [hidden]="hasOtherUtilityData">*</span> -->
             </a>
         </li>
```

Comment:
This whole page is incorrect. I recently fixed this in a PR.  [class.red]="xyz" should be on the list element. Not the anchor.

![Screen Shot 2021-04-20 at 9 48 20 AM](https://user-images.githubusercontent.com/57914806/115407010-b0152a00-a1bd-11eb-9c5a-15ea0beb104c.png)

Reply:
Merging error. Wondering why it was yelling at me. My bad. Fixed